package/MDAnalysis/analysis/rms.py

```diff
@@ -351,7 +351,7 @@ def __init__(self, traj, reference=None, select='all',
         self.ref_atoms = self.reference.select_atoms(*self.select['reference'])
         self.traj_atoms = self.universe.select_atoms(*self.select['mobile'])
         if len(self.ref_atoms) != len(self.traj_atoms):
-            logger.exception()
+            logger.exception('SelectionError: ref_atoms, traj_atoms unequal')
```

Comment:
(I think you can also leave the message for `logger.exception` empty and it will show the stack trace.)  But otherwise make sure that they have the same message, e.g.
```python
errmsg = "Reference and trajectory atom ... N_ref={0:d}, N_traj={1:d}".format(...)
logger.exception(errmsg)
raise SelectionError(errmsg)
```